docs/book/programs/lsm.md

```diff
@@ -87,7 +87,7 @@ GRUB_CMDLINE_LINUX="lsm=[YOUR CURRENTLY ENABLED LSMs],bpf"
 Then rebuilding the grub configuration with:
 
 ```console
-grub-mkconfig -o /boot/grub/grub.cfg
+sudo update-grub2
```

Comment:
I don't see this command on openSUSE and Arch. I think we should mention both commands and ask people to double check which one is available for them.

Reply:
Thanks for reviewing and As per your suggestion. I have modified the document.